charts/internal/gvisor/templates/daemonset.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: containerd-gvisor
+  namespace: kube-system
+  labels:
+    app.kubernetes.io/name: containerd-gvisor
+    helm.sh/chart: containerd-gvisor
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: containerd-gvisor
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: containerd-gvisor
+    spec:
+      hostPID: true
+      hostIPC: true
+      nodeSelector:
+{{ toYaml .Values.config.nodeSelector | indent 8 }}
+      containers:
+      - name: container-runtime-gvisor-containerd
+        image: "docker.io/nimrodoron/runtime-gvisor:latest"
```

Comment:
I guess this should be the image of the gardener-extension-provider-gvisor, right? :)

Instead of hard-coding the image here, please add it to `charts/images.yaml` and inject it via the image vector (like in gardener and all other extensions).

Reply:
Also, instead of specifying `latest`, please specify a dedicated version.

Reply:
Why are we using one docker image that serves two purposes? 
1) gvisor extension to reconcile the ContainerRuntime
2) Image for the DeamonSet to install the container runtime binaries on the host

@rfranzke you suggested that in the previous comment, is this how it is done in similar cases in Gardener? 

I would split that up into two separate images.

Reply:
No strong opinion here. It’s also okay for me if this it put in a separate image. In the end, it’s anyways only used to copy over the gvisor binary to the host.

Reply:
I think one image will be easier to maintain.

Reply:
OK fine with that, thanks!